src/main/java/com/shop/console/utils/Menu.java

```diff
@@ -0,0 +1,42 @@
+package com.shop.console.utils;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+
```

Comment:
I can't see logic for scanning categories. One of the requirements is:
"The list of categories should be created based on the list of classes in the
target package. Every class in this package should relate to a single category
and have a constant with a category name. Accordingly, it should be a feature
to add a new category by adding a new class into the package and restarting
the application."

Try to think, how to design it